README.md

```diff
@@ -20,54 +47,46 @@ You can enable and configure this plugin in your `gatsby-config.js` file.
 
 ### Basic usage
 
+First, you need to configure the `STRAPI_API_URL` and the `STRAPI_TOKEN` environment variables. We recommend using [`dotenv`][https://github.com/motdotla/dotenv] to expose these variables.
+
+> Make sure to create a full-access API TOKEN in Strapi
```

Comment:
I don't think the quote (`>`) is appropriate here, and it makes it less readable on GitHub. Also I think we should link to [our docs on how to create API tokens](https://docs.strapi.io/user-docs/latest/settings/managing-global-settings.html#managing-api-tokens), especially since this is a new feature and people may not know about it